lib/dal/tests/integration_test/internal/pkg.rs

```diff
@@ -627,8 +627,8 @@ async fn test_install_pkg(ctx: &DalContext) {
         .data(
             FuncSpecData::builder()
                 .name("si:truthy")
-                .display_name("truth")
-                .description("it returns true")
+                .display_name("truth".to_owned())
```

Comment:
What happened here to make this necessary?

Reply:
We were removing the option in the setter, which made it annoying to add an option to it. Since we were building the spec from the pkg, and the field in the package also is an option, it doesn't make sense to have to if let in the field for the package to set an optional to the spec.